data_model/src/lib.rs

```diff
@@ -389,6 +385,28 @@ pub enum IdentifiableBox {
     Role(Box<role::Role>),
 }
 
+impl RegisteredWith for trigger::Trigger<FilterBox> {
+    type RegisteredWith = Self;
+}
+
+impl IdentifiableBox {
+    fn contains(&self, content: &str) -> bool {
```

Comment:
why do we need this? I suppose this will be revealed when more stuff is implemented

Reply:
This is necessary for the predicates. I havent pushed the function that uses it yet, since I'm still playing around with the API.